### PR TITLE
Resolve #217

### LINKED ISSUE #217: [FEATURE] Add command which sends message to roles

### PR DESCRIPTION
**Pull request by Denio21**

### CODE REVIEW

File: languages/en.json [hunk 1]
```diff
@@ -423,5 +425,14 @@
...
+    "builder.mute.add.ok": "Builder is complete. Member is muted",
+
+    "builder.senddm.title.title": "Title of direct message",
+    "builder.senddm.title.help": "Enter title of direct message",
+    "builder.senddm.title": "Direct messages sender",
+    "builder.senddm.role.title": "Role for direct message",
+    "builder.senddm.role.help": "Enter name of the role, which you want to be selected for sending direct message",
+    "builder.senddm.message.title": "Content of direct message",
+    "builder.senddm.message.help": "Enter content, which will be sent in direct message",
+    "builder.senddm.ok": "Builder je complete. The message was sent"
```
Comment (cajthamlmatej) [inline languages/en.json]: "je"? What does it mean?

File: src/commands/SendDm.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        ], (values) => {
+            logger.info("User " + message.member.displayName + " sent dms to " + values["role"] + ". Content of message is \"" + values["message"] + "\"");
+
+            const dmembed = new Discord.RichEmbed()
+                .setTitle("🧬 | " + values["title"])
+                .setColor(Config.getColor("SUCCESS"))
+                .setDescription(values["message"])
+                .setFooter(message.member.displayName, message.author.avatarURL);
+
+            message.channel.send(dmembed);
```
Comment (cajthamlmatej) [inline src/commands/SendDm.js]: As I said, disable this.

File: src/commands/SendDm.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    getAliases() {
+        return ["dm"];
+    }
+
+    getRoles() {
+        return ["moderator"];
+    }
+
+    init(bot) {
+        this.client = bot.client;
```
Comment (cajthamlmatej) [inline src/commands/SendDm.js]: Unused variable

File: languages/en.json (hunk 1 above)
Comment (Denio21) [reply languages/en.json]: I thought i have seen that. Fixing it in second.

File: src/commands/SendDm.js (hunk 2 above)
Comment (Denio21) [reply src/commands/SendDm.js]: Sorry, i forgot.

File: src/commands/SendDm.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        return ["dm"];
+    }
+
+    getRoles() {
+        return ["moderator"];
+    }
+
+    init(bot) {
+        this.eventModule = bot.modules.eventmodule;
+        this.channel = bot.client.channels.find(ch => ch.id == Config.get("channels.annoucement"));
```
Comment (cajthamlmatej) [inline src/commands/SendDm.js]: Unused variable

File: src/commands/SendDm.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
+const Command = require("./Command");
+const Discord = require("discord.js");
+const CommandBuilder = require("../CommandBuilder");
+const Translation = require("../Translation");
+const Config = require("../Config");
```
Comment (cajthamlmatej) [inline src/commands/SendDm.js]: Going to be unused

File: src/commands/SendDm.js (hunk 4 above)
Comment (Denio21) [reply src/commands/SendDm.js]: what do u mean exactly?

**Pull request merged**